contributor-dashboard-legacy/src/components/ForkRibbon.js

```diff
@@ -62,7 +63,11 @@ const Ribbon = styled.a`
 
 		/* Set the text properties */
 		color: #fff;
-		font: 700 1em "Helvetica Neue", Helvetica, Arial, sans-serif;
+		font:
```

Comment:
interesting, tabs vs spacing. I would imagine prettier complaining about this

Reply:
Didn't mean this line specifically, not sure why I put the comment here. I'm just surprised to see tabs everywhere. Doesn't prettier indent with spaces by default?